components/calendly/calendly.app.js

```diff
@@ -0,0 +1,47 @@
+const axios = require("axios");
+
+module.exports = {
+  type: "app",
+  app: "calendly",
+  methods: {
+    _getHeader() {
```

Comment:
Consider renaming this to `_getAuthHeader()` to be explicit about the type of header